crt_portal/static/sass/custom/complaint.scss

```diff
@@ -0,0 +1,67 @@
+$complaint-header-min-height: 200px;
+
+.complaint-header {
+  min-height: $complaint-header-min-height;
+  width: 100%;
+
+  & .close-record {
+    margin-right: 0;
+    // Matches h2 margin-top value
+    margin-top: 1.66rem;
+    text-decoration: none;
+
+    &:visited {
+      color: $blue-50;
+    }
+  }
+}
+
+.complaint-body {
+  transform: translateY(calc(-120% + #{$complaint-header-min-height}));
```

Comment:
@el-mapache Curious what this line of code is doing — I don't know how to read it.

Reply:
Yeah this got a little muddled. I have a simpler calculation that can express the offset we want here. I'm using the `transform: translate` syntax because I think it conveys the intention that this element is going to be _placed_ somewhere on the page better than `margin-top`!

Reply:
new calc is 
```
$complaint-body-offset: calc(
  -#{$twenty-percent-mult} * #{$complaint-header-min-height}
);
```

Reply:
And if you are asking specifically about the `transform: translateY` call, is basically just means: move the element `n` units (`px`, `%`, etc) vertically along a 2D plane, relative the element's current position in the browser window. 

So, if the element is `300px` from the top of the `window`, and you apply a `translateY(40px)`, the element will move down an additional 40px to 340px offset from the window.

Reply:
@el-mapache I understand much better now! I like the distinction in intention you drew between `translate` and `margin`. Thanks!